### ISSUE TITLE
Super cool new battleships, in 2D side view!

### ISSUE THREAD

**Issue by maximusvladimir** [JPII/NavalBattle#147]

**Comment by ThomasGaubert** [comment]
Can you explain this one?

**Comment by maximusvladimir** [comment]
The game will now render in side view mode during an attack with that super cool ship that y'all sent me.

**Comment by ThomasGaubert** [comment]
Oh cool, I see.

It won't become annoying or slow gameplay (the flow of the game, not necessarily the performance) hopefully, right?

**Comment by maximusvladimir** [comment]
It will happen at the end of each turn, and all the health and money has been adjusted to the proper levels. Simply clicking will end it.

**Comment by ThomasGaubert** [comment]
Oh okay, that sounds like a better idea! Pokemon style I guess?

**Comment by maximusvladimir** [comment]
I will try to make a demo later this week. It will be somewhat like the titlescreen, where there are ships and waves with a side view. Basically the tomahawks/missiles/whatever will travel from ship to ship.

**Comment by abauer** [comment]
oh ok, as long as it doesn't impact game-play to much and from previous comments i assume its skip-able.

**Comment by abauer** [comment]
and this is not top priority again this is a feature and is low priority

**Comment by maximusvladimir** [comment]
It is very skip-able. All you have to do is sign a 101-page contract, then click the mouse button anywhere on the screen. ;)

**Comment by abauer** [comment]
ok cool, just my type of cut-scene

**Comment by maximusvladimir** [comment]
Yeah, okay I don't think this is going to be possible to do anymore.
1. Requires too much CPU.
2. Ain't nobody got enough time to implement that.

**Comment by ThomasGaubert** [comment]
Possibly something to look into for post-launch.

**Comment by maximusvladimir** [comment]
Hahaha I can do airplanes.

![gif](http://24.media.tumblr.com/tumblr_ljv7cl4MLp1qa51o3o1_500.gif)

**Comment by abauer** [comment]
darn... though i don't think this is vital to the game

**Comment by abauer** [comment]
so are we going to close / move this one?

**Comment by maximusvladimir** [comment]
Probably move it.

**Comment by abauer** [comment]
moved to 0.9

**Comment by abauer** [comment]
i don't think this is vital to the game and think it should be moved.

**Comment by maximusvladimir** [comment]
Agreed.

**Comment by ThomasGaubert** [comment]
Removed milestone.
